src/app/zones/views/ZoneDetails/ZoneDetails.test.tsx

```diff
@@ -91,13 +85,8 @@ describe("ZoneDetails", () => {
         </MemoryRouter>
       </Provider>
     );
-    expect(
-      wrapper
-        .findWhere(
-          (node) =>
-            node.type() === "button" && node.text() === Labels.EditButton
-        )
-        .exists()
-    ).toBe(false);
+
+    const editButtons = screen.queryAllByRole("button", { name: "Edit" });
+    editButtons.forEach((button) => expect(button).not.toBeInTheDocument());
```

Comment:
Here the test would pass if we changed the button label in the component but didn't update it here.